LowerBodyModule.py

```
@@ -33,7 +34,7 @@ def lowerBody():
                 dir = 1
         if per == 0:
             cv2.rectangle(img, (600, 100), (650, 370), (0, 0, 255), thickness=2)
-            cv2.rectangle(img, (600, int(height)), (650, 370), (0, 0, 255), thickness=cv2.FILLED)
+            cv2.rectangle(img, (600, int(height)), (650, 370), (0, 0, 255,23242343), thickness=cv2.FILLED)
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)